modules/vstudio/vs2005_dotnetbase.lua

```diff
@@ -687,7 +687,6 @@
 		end
 	end
 
-
```

Comment:
I'm fairly certain that there's normally two lines between functions. This also applies to the new lines below for `dotnetbase.csversion`.